src/custom-surf/pages/ServiceTickets.tsx

```diff
@@ -365,6 +365,25 @@ class ServiceTickets extends React.PureComponent<IProps, IState> {
                                     >
                                         {renderIsoDatetime(ticket.last_update_time, true)}
                                     </td>
+                                    <td>
+                                        {ticket.process_state === ServiceTicketProcessState.NEW && (
+                                            <EuiButtonIcon
+                                                id={`tickets.table.${ticket._id}.delete`}
+                                                iconType="trash"
+                                                aria-label={intl.formatMessage({ id: "tickets.table.delete" })}
+                                                onClick={(
+                                                    e: React.MouseEvent<
+                                                        HTMLButtonElement | HTMLAnchorElement,
+                                                        globalThis.MouseEvent
+                                                    >
+                                                ) => {
+                                                    this.context.customApiClient
```

Comment:
create a function outside the html for this and use async await

Reply:
Done :+1: